src/openklant/components/klantinteracties/api/tests/test_klantcontacten.py

```diff
@@ -1652,19 +1659,213 @@ def test_filter_onderwerpobject_by_klantcontact_filters(self):
                 self.assertEqual(len(data["results"]), 1)
                 self.assertEqual(data["results"][0]["uuid"], str(obj.uuid))
 
-    def test_destroy_onderwerpobject(self):
-        onderwerpobject = OnderwerpobjectFactory.create()
-        detail_url = reverse(
+    def test_destroy_with_cascade_true(self):
```

Comment:
Could you also add a test with `cascade=true` where there are remaining klantcontacten (so the status code is 200)?